tests/conftest.py

```diff
@@ -20,4 +27,10 @@ def pytest_collection_modifyitems(config, items):
     else:
         for item in items:
             if "largemodel" in item.keywords:
-                item.add_marker(skip_largemodel)
\ No newline at end of file
+                item.add_marker(skip_largemodel)
+
+    if config.getoption("--legacy-unstructured"):
```

Comment:
This is my attempt to run integration test suite on old structured index. This kinda works, but it takes about 40min and it also runs tests for structured indexes which are not needed. I will probably revert this commit and find another way to test legacy unstructured index (maybe using compatibility test suite in the future?)